2018-09-19 21:27:20 - root - ERROR: - AI | 1 | "Yang Yang" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Yang%20Yang%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 1
2018-09-19 21:28:20 - root - ERROR: - AI | 1 | "Yang Yang" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Yang%20Yang%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 1
2018-09-19 21:28:31 - root - ERROR: - AI | 1 | "Yang Yang" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Yang%20Yang%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 2
2018-09-19 21:28:41 - root - ERROR: - AI | 1 | "Yang Yang" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Yang%20Yang%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 3
2018-09-19 21:28:58 - root - ERROR: - AI | 1 | "Yang Yang" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Yang%20Yang%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 1
2018-09-19 21:29:08 - root - ERROR: - AI | 1 | "Yang Yang" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Yang%20Yang%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 2
2018-09-19 21:29:17 - root - ERROR: - AI | 1 | "Yang Yang" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Yang%20Yang%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 3
2018-09-19 21:29:26 - root - ERROR: - AI | 1 | "Yang Yang" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Yang%20Yang%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 4
2018-09-19 21:29:37 - root - ERROR: - AI | 1 | "Yang Yang" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Yang%20Yang%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 5
2018-09-19 21:29:49 - root - ERROR: - AI | 1 | "Yang Yang" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Yang%20Yang%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 6
2018-09-19 21:30:02 - root - ERROR: - AI | 2 | "Stephen Grossberg" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Stephen%20Grossberg%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 1
2018-09-19 21:30:10 - root - ERROR: - AI | 2 | "Stephen Grossberg" | HTTPSConnectionPool(host='scholar.google.com', port=443): Max retries exceeded with url: /citations?view_op=search_authors&hl=en&mauthors=%22Stephen%20Grossberg%22 (Caused by ProxyError('Cannot connect to proxy.', ConnectionResetError(10054, '远程主机强迫关闭了一个现有的连接。', None, 10054, None))) | trries: 2
